src/components/DashboardsCard/DashboardsCard.tsx

```diff
@@ -93,9 +93,22 @@ const Dashboards = ({
   opts?: DashboardCardOpts;
 }) => {
   const grafanaApi = useApi(grafanaApiRef);
-  const { value, loading, error } = useAsync(
-    async () => await grafanaApi.dashboardsByTag(tagSelectorFromEntity(entity)),
-  );
+  const { value, loading, error } = useAsync(async () => {
+    const grafanaDashboards = await grafanaApi.dashboardsByTag(tagSelectorFromEntity(entity));
+    const grafanaAnnotation = entity.metadata.annotations?.["grafana/links"] || '';
+    const grafanaUrls = grafanaAnnotation.split(';');
+    grafanaUrls.forEach(g => {
+      const parts = g.split('::', 2);
+      grafanaDashboards.push({
+        id: Date.now() + Math.random(),
+        url: parts[1],
+        title: parts[0],
+        folderUrl: "",
+        folderTitle: ''
+      });
+    });
```

Comment:
This seems very specific to how we use the plugin, can you try to instead pass a hook to this component that can add additional dashboards based on the entity?  Then we could move this code (or similar code) to our Backstage installation and leave more flexibility to this plugin.

Reply:
Okay, I get the idea. I will try this solution then